### ISSUE TITLE
error NJS-098 with colons in sql comments since version 6.0.0

### ISSUE THREAD

**Issue by ritschwumm** [oracle/node-oracledb#1561]
using the new thin driver, colons in comments sometimes seem to be taken as introducing a bind parameter:

```sql
select
  1
from
  -- :
  dual
where
  -- :
  1=1
```

results in 
```
NJS-098: 1 positional bind values are required but 0 were provided
```

interestingly, neither of the two colons can be removed, or the error disappears

**Comment by sharadraju** [comment]
Thanks @ritschwumm , Able to reproduce the issue. We will look into it

**Comment by sharadraju** [comment]
The patch in `lib/thin/statement.js` for this issue is now available as indicated by the link above. To apply this patch, simply copy this file to your lib folder and replace the older version.
This will be incorporated in the 6.0.1 release.

**Comment by sharadraju** [comment]
Closing the issue as it is fixed in 6.0.1 release.

**Comment by ritschwumm** [comment]
thanks a lot :)
